dandischema/metadata.py

```diff
@@ -101,19 +101,34 @@ def publish_model_schemata(releasedir: str) -> Path:
     return vdir
 
 
+def _validate_obj_json(data, schema, missing_ok=False):
+    validator = jsonschema.Draft7Validator(
+        schema, format_checker=jsonschema.draft7_format_checker
+    )
+    error_list = []
+    for error in sorted(validator.iter_errors(data), key=str):
+        if missing_ok and "is a required property" in error.message:
+            continue
+        error_list.append([error])
+    if error_list:
+        raise ValueError(error_list)
```

Comment:
Echoing prior discussion, Let's at least define our own `class ValidationError(ValueError)` may be?

Reply:
do you want to do it now? or later?

Reply:
If no objections, will push that change in 5 minutes. Better not to delay ;-)

Reply:
Later we would need to reactor anyways though

Reply:
ok then go for it.

Reply:
any other changes you plan, or should i merge?